domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -2,12 +2,13 @@
 using API.Services;
 using Microsoft.AspNetCore.Mvc;
 using System.ComponentModel.DataAnnotations;
+using API.Services.OidcProviders;
 
 namespace API.Controllers;
 
 
 [ApiController]
-public class AuthController
+public class AuthController : ControllerBase
 {
     readonly ILogger<AuthController> logger;
```

Comment:
Should we use this logger on incomming request or can we remove it? @duizer

Reply:
If `logger` is not used in the Controller, then you can remove it. We do not manually log information about the request (or the response) using `logger` in other projects.